### PR TITLE
refactor(common-entity-webapp-ext): move to packages

### PR DESCRIPTION
**Pull request by Kaskyi**
Move ext-common-entity to packages.
Make independent module.

Code module location is `@xm-ngx/common-entity-webapp-ext`
if you have any `../../ext-common-entity/..` make sure to update them to `@xm-ngx/common-entity-webapp-ext`.

### CODE REVIEW

File: packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts [hunk 1]
```diff
@@ -46,10 +45,10 @@ export class EntityWidgetComponent implements OnInit, OnDestroy, XmDynamicWidget
 
     public ngOnInit(): void {
         this.showLoader = true;
-        if (DEBUG_INFO_ENABLED) {
-            console.info('DBG entity  e=%o', this.xmEntity);
-            console.info('DBG spec  e=%o', this.spec);
-        }
+        // if (DEBUG_INFO_ENABLED) {
```
Comment (sergeysenja1992) [inline packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts:48]: ?

Comment (Kaskyi) [reply packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts]: done

File: packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts [hunk 2]
```diff
@@ -39,17 +38,18 @@ export class EntityWidgetComponent implements OnInit, OnDestroy, XmDynamicWidget
...
     }
 
     public ngOnInit(): void {
         this.showLoader = true;
-        if (DEBUG_INFO_ENABLED) {
-            console.info('DBG entity  e=%o', this.xmEntity);
-            console.info('DBG spec  e=%o', this.spec);
-        }
+
+        this.logger.debug(`entity=${JSON.stringify(this.xmEntity)}`);
```
Comment (sergeysenja1992) [inline packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts:50]: this.logger.debug write wrong line of code in console

**Review by sergeysenja1992 — CHANGES_REQUESTED** [review]

File: packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts [hunk 3]
```diff
@@ -163,13 +163,15 @@ export class EntityWidgetComponent implements OnInit, OnDestroy, XmDynamicWidget
...
                 tap((entity) => this.backLinkSpecs = this.getBackLinkSpecs(entity.typeKey)),
                 tap(() => this.defineUiConfig()),
-                tap(() => this.linkSpecs$.next(
-                    this.xmEntitySpec && this.xmEntitySpec.links ?
-                        this.xmEntitySpec.links.map((spec) => this.addInterfaceSpec(spec, this.entityUiConfig)) : [],
-                ),
+                tap(() =>
+                    this.linkSpecs$.next(
+                        this.xmEntitySpec && this.xmEntitySpec.links
+                            ? this.xmEntitySpec.links.map((spec) => this.addInterfaceSpec(spec, this.entityUiConfig))
```
Comment (sergeysenja1992) [inline packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts:172]: before formatting better

**Review by grankovsky — APPROVED** [review]

File: packages/common-entity-webapp-ext/entity-fab-actions/entity-fab-actions.component.ts [hunk 4]
```diff
@@ -15,9 +15,15 @@ import {
     XmEntitySpecWrapperService,
 } from '@xm-ngx/entity';
 import { FunctionCallDialogComponent } from '@xm-ngx/entity';
-import { XM_EVENT_LIST } from '../../../xm.constants';
 import { pluck, takeUntil } from 'rxjs/operators';
 
+enum XM_EVENT_LIST {
```
Comment (denisogolubev) [inline packages/common-entity-webapp-ext/entity-fab-actions/entity-fab-actions.component.ts:20]: It's better to name an enum using the UpperCase naming convention, following the TS official documentation:
https://www.typescriptlang.org/docs/handbook/enums.html

**Review by denisogolubev — APPROVED** [review]

File: packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts (hunk 2 above)
Comment (Kaskyi) [reply packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts]: done

File: packages/common-entity-webapp-ext/entity-fab-actions/entity-fab-actions.component.ts (hunk 4 above)
Comment (Kaskyi) [reply packages/common-entity-webapp-ext/entity-fab-actions/entity-fab-actions.component.ts]: done

File: packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts (hunk 3 above)
Comment (Kaskyi) [reply packages/common-entity-webapp-ext/entity-widget/entity-widget.component.ts]: done

**Pull request closed without merging**